week19/restock-shopping-cart_starter/cart.jsx

```diff
@@ -186,10 +146,10 @@ const Products = (props) => {
   const restockProducts = (url) => {
     // ADD CODE HERE!!!
     console.log('restocking products...');
-    apiCall(url);
-    console.log(data);
-    // TO DO: rename data to something more helpful so we can call something.data.map()
-    let newItems = data.data.map((newItem) => {
+    doFetch(url);
```

Comment:
we pass the url in to doFetch (aka setUrl in useDataApi) because useEffect will update the values whenever the url is updated